src/backend/src/game/game.controller.ts

```diff
@@ -36,29 +57,40 @@ export class GameController {
       throw new BadRequestException('Player does not exist');
     }
 
-    const round = await this.roundsRepository.findOne(roundId);
+    let round = await this.roundsRepository.findOne(roundId);
```

Comment:
Whats the differrence between this round and the one below? Why Can't we use the one below to begin with?

Reply:
I think the idea is we add this round to the grab first, then we find the updated round. Not sure if we can just do it all the first time?